v21.1/build-a-nodejs-app-with-cockroachdb-prisma.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Build a Simple CRUD Node.js App with CockroachDB and Prisma Client
+summary: Learn how to use CockroachDB from a simple CRUD application that uses the Prisma Client ORM.
+toc: true
+twitter: false
+referral_id: docs_node_prisma
+---
```

Comment:
@ericharmeling, please open a new PR to add `docs_area`. Please also check other recent PRs where you added new pages, or changed the location of existing pages.